client/my-sites/customer-home/locations/upsells/index.jsx

```diff
@@ -10,41 +9,45 @@ import { connect } from 'react-redux';
  */
 import isSiteChecklistComplete from 'state/selectors/is-site-checklist-complete';
 import isUnlaunchedSite from 'state/selectors/is-unlaunched-site';
-import { getSiteOption } from 'state/sites/selectors';
 import StatsBanners from 'my-sites/stats/stats-banners';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
 
-const Upsells = ( {
-	isChecklistComplete,
-	isEstablishedSite,
-	siteId,
-	siteIsUnlaunched,
-	siteSlug,
-} ) => {
+const cardComponents = {
+	'home-banner-legacy-stats-banners': StatsBanners,
+};
+
+const Upsells = ( { cards, primaryButton, siteId, slug } ) => {
+	const statsBannersProps = {
+		primaryButton,
+		siteId,
+		slug,
+	};
```

Comment:
A janitorial note for later, but we should aim to have all cards handle their own data, so we don't end up querying for information we don't need in the location components.

Reply:
Yes, I started down that road, but it all just got way wider than the point of the PR. Followup cleanup will be a good idea (like the strings you mentioned too).